src/libraries/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/ComInterop/ComEventsSink.Extended.cs

```diff
@@ -15,6 +16,7 @@ private void Initialize(object rcw, Guid iid)
             Advise(rcw);
         }
 
+        [RequiresUnreferencedCode(Microsoft.CSharp.RuntimeBinder.Binder.TrimmerWarning)]
```

Comment:
```suggestion
        [RequiresUnreferencedCode(Binder.TrimmerWarning)]
```

Reply:
there's a using statement missing in there for Microsoft.CSharp.RuntimeBinder but will add it